r/vignettes/dev-docs.Rmd

```diff
@@ -0,0 +1,427 @@
+---
```

Comment:
I might change the file name to something like `developing.Rmd`, makes for a better URL IMO.